models/_sector/nft/trades/solana/platforms/tensorswap_v2_solana_trades.sql

```diff
@@ -0,0 +1,139 @@
+{{
+    config(
+        schema = 'tensorswap_v2_solana'
+        , tags = ['dunesql']
+        , alias = alias('trades')
+        ,materialized = 'incremental'
+        ,file_format = 'delta'
+        ,incremental_strategy = 'merge'
+        ,incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+        ,unique_key = ['project','trade_category','outer_instruction_index','inner_instruction_index','account_merkle_tree','leaf_id','tx_id']
```

Comment:
in the refactor, we should try to make it so all projects have the same unique keys, even if that means forcing default values on columns in specific models